testing/e2e/suites/utils.go

```diff
@@ -62,3 +63,11 @@ func upsert(ctx context.Context, c client.Client, obj client.Object) error {
 	lgr.Info("object updated")
 	return nil
 }
+
+func getTestSecret(name, namespace string) *v1alpha1.Secret {
```

Comment:
I don't understand the point of this helper function. It doesn't make anything simpler or abstract anything out